index.js

```diff
@@ -0,0 +1,10 @@
+const navToggle = document.querySelector('#navToggle');
```

Comment:
- [ ] You have not implemented any code for the popup window. Kindly double check your dashboard before submission.